app/models/candidate_pool_application.rb

```diff
@@ -91,4 +91,22 @@ def self.filter_by_funding_type(scope, filters)
       scope.where(course_funding_type_fee: true)
     end
   end
+
+  def self.closed?
+    timetable = RecruitmentCycleTimetable.current_timetable
+    return true if timetable.between_cycles?
+
+    now = Time.zone.now
+    cycle_start = DateTime.new(
+      now.year,
+      timetable.apply_opens_at.month,
+      timetable.apply_opens_at.day,
+    )
+
+    now.between?(cycle_start, open_at)
+  end
+
+  def self.open_at
+    DateTime.new(Time.zone.now.year, 11, 20)
```

Comment:
Since we want to open generally after the first wave of candidates are made inactive, could this be: 
```suggestion
    days = CandidateInterface::InactiveDateCalculator::STANDARD_DAYS_UNTIL_INACTIVE + 1
    days.business_days.after(timetable.apply_opens_at).end_of_day
```

Or `6.weeks.after(timetable.apply_opens_at).end_of_day` 

Both give us 19 November 23:59:59 2025.